source/compiler/qsc_codegen/src/qir.rs

```diff
@@ -728,8 +728,8 @@ fn get_module_metadata(program: &rir::Program) -> String {
                     let name = "int_computations";
                     writeln!(
                         flags,
```

Comment:
probably worth a note of what 5 means. Or introduce an enum that impls `From<i64>`.